src/Entities/Prescriptions.php

```diff
@@ -0,0 +1,841 @@
+<?php
+/**
```

Comment:
remember to only include the entities that are used in this PR (it seems like there are more than the last PR)

Reply:
removed

Reply:
Put back because this is used int he PrescriptionsRepository class